nservicebus/dependency-injection/index_ownership_core_[7.2,).partial.md

```diff
@@ -0,0 +1,23 @@
+NServiceBus automatically registers and invokes message handlers, sagas, and other user provided extension points using a dependency injection container.
+
+NServiceBus supports two modes of operation, *internally managed* and *externally managed* container.
+
+## Internally managed mode
+
+In *internally managed* mode, NServiceBus manages the entire lifecycle of the container from registration, component resolution to disposal.
+
+include: internallymanagedcontainer
+
+## Externally managed mode
+
+In *externally managed* mode, NServiceBus registers its components in the container but does not own the container's lifecycle. The container is passed by the user via two different abstractions, one for the registration phase (`IConfigureComponents`) and one for the resolution phase (`IBuilder`).
```

Comment:
```suggestion
In *externally managed* mode, NServiceBus registers its components in the container but does not own the container's lifecycle. The container is provided by the user in two phases, one for registration (`IConfigureComponents`) and one for resolution (`IBuilder`).
```

Reply:
@andreasohlund did you decide against this suggestion? I see you didn't commit it.